wrapper/tests/test_v2v_args.py

```diff
@@ -46,6 +46,17 @@ def assert_has_args(self, arg_list, expected, msg=None):
         'install_drivers': False,
         'output_format': 'raw',
         'insecure_connection': False,
+
+        'luks_keys_files': [
+            {
+                'device': '/dev/sda1',
+                'filename': '/tmp/luks/sda1',
+            },
+            {
+                'device': '/dev/sda2',
+                'filename': '/tmp/luks/sda2',
+            }
+        ]
```

Comment:
Create a new dictionary with the LUKS specific lines. I.e. copy VDDK_RHV and extend it with the above lines.

Reply:
Should it still be the case, now that I moved it to `wrapper/tests/test_v2v_args.py` ?

Reply:
Please pick this commit 745fe83f6597. It will fix the tests.